packages/quick_actions/quick_actions/README.md

```diff
@@ -19,23 +20,40 @@ Initialize the library early in your application's lifecycle by providing a
 callback, which will then be called whenever the user launches the app via a
 quick action.
 
+<?code-excerpt "main.dart (Initialize)"?>
 ```dart
-final QuickActions quickActions = const QuickActions();
-quickActions.initialize((shortcutType) {
-  if (shortcutType == 'action_main') {
-    print('The user tapped on the "Main view" action.');
-  }
-  // More handling code...
+const QuickActions quickActions = QuickActions();
+quickActions.initialize((String shortcutType) {
+  setState(() {
+    shortcut = shortcutType;
+  });
 });
 ```
 
 Finally, manage the app's quick actions, for instance:
-
+<?code-excerpt "main.dart (SetShortcuts)"?>
 ```dart
 quickActions.setShortcutItems(<ShortcutItem>[
-  const ShortcutItem(type: 'action_main', localizedTitle: 'Main view', icon: 'icon_main'),
-  const ShortcutItem(type: 'action_help', localizedTitle: 'Help', icon: 'icon_help')
-]);
+  // NOTE: This first action icon will only work on iOS.
+  // In a real world project keep the same file name for both platforms.
```

Comment:
We either need to change the example app not to have this behavior, or use an excerpt file; this is far more confusing than the snippet that it's replacing.

Reply:
@stuartmorgan Removed this & added platform-specific checks in the example code.